api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -217,7 +92,7 @@ private DbUser initializeUserIfNeeded() {
     // the CDR); we will probably need a job that deactivates accounts after some period of
     // not accepting the terms of use.
 
-    DbUser dbUser = initializeUserIfNeeded();
+    DbUser dbUser = profileService.initializeUserIfNeeded();
     profileAuditor.fireLoginAction(dbUser);
     return getProfileResponse(dbUser);
```

Comment:
There's no real need to have a local DbUser object but this pattern is everywhere.  Deferring updating this to future PRs.